### PR TITLE
IMPRO-1815 sleetacc and rainacc bounds for ECC

### PR DESCRIPTION
**Pull request by MoseleyS**
Adds bounds for data named `lwe_thickness_of_sleetfall_amount` and `thickness_of_rainfall_amount` (identical to values for snow) so that engl sleet chain can extract a 50th percentile.

Testing:
 - [x] Ran tests and they passed OK
 - [x] Added new tests for the new feature(s) - nothing to add.

### CODE REVIEW

File: improver/ensemble_copula_coupling/constants.py [hunk 1]
```diff
@@ -71,6 +71,8 @@
     "relative_humidity": Bounds((0, 1.2), "1"),
     "lwe_thickness_of_precipitation_amount": Bounds((0, 0.5), "m"),
     "lwe_thickness_of_precipitation_amount_in_vicinity": Bounds((0, 0.5), "m"),
+    "thickness_of_rainfall_amount": Bounds((0, 0.5), "m"),
+    "lwe_thickness_of_sleetfall_amount": Bounds((0, 0.5), "m"),
```
Comment (bayliffe) [inline improver/ensemble_copula_coupling/constants.py:75]: Do you fancy sorting out the grouping of all of these whilst you are making changes? The list is **not**:
- alphabetical
- sorted by diagnostic type
- sorted by matching thresholds

Perhaps thicknesses with thicknesses, alphabetical in their group, rates with rates, alphabetical in their group etc?
- in any semblance of an order.

**Review by bayliffe — APPROVED** [review]

File: improver/ensemble_copula_coupling/constants.py (hunk 1 above)
Comment (MoseleyS) [reply improver/ensemble_copula_coupling/constants.py:65]: Like this?

File: improver/ensemble_copula_coupling/constants.py [hunk 2]
```diff
@@ -48,39 +48,48 @@
...
-    "rainfall_rate": Bounds((0, 128.0), "mm h-1"),
-    "rainfall_rate_in_vicinity": Bounds((0, 128.0), "mm h-1"),
-    "relative_humidity": Bounds((0, 1.2), "1"),
+    # Precipitation amount
     "lwe_thickness_of_precipitation_amount": Bounds((0, 0.5), "m"),
     "lwe_thickness_of_precipitation_amount_in_vicinity": Bounds((0, 0.5), "m"),
+    "lwe_thickness_of_sleetfall_amount": Bounds((0, 0.5), "m"),
     "lwe_thickness_of_snowfall_amount": Bounds((0, 0.5), "m"),
+    "thickness_of_rainfall_amount": Bounds((0, 0.5), "m"),
+    "air_pressure_at_sea_level": Bounds((86000, 108000), "Pa"),
```
Comment (gavinevans) [inline improver/ensemble_copula_coupling/constants.py:68]: Should `air_pressure_at_sea_level` be moved down into the "Others" group?

**Review by gavinevans — CHANGES_REQUESTED** [review]
Thanks for this tidy-up, @MoseleyS 👍 

I've added one minor comment.

File: improver/ensemble_copula_coupling/constants.py (hunk 2 above)
Comment (MoseleyS) [reply improver/ensemble_copula_coupling/constants.py]: Definitely! Well done for spotting my deliberate mistake, to put a positive spin on it.

**Review by gavinevans — APPROVED** [review]

**Pull request merged**